lib/grammers-client/src/lib.rs

```diff
@@ -46,3 +46,9 @@ pub(crate) mod utils;
 
 pub use client::{Client, Config, InitParams, SignInError};
 pub use types::{button, reply_markup, ChatMap, InputMessage, Update};
+
+pub use grammers_crypto;
+pub use grammers_mtproto;
+pub use grammers_mtsender;
+pub use grammers_session;
+pub use grammers_tl_types;
```

Comment:
The ultimate goal is for `grammers-client` to never expose or accept raw types as input, so I'm not sure we should always export this either.

Maybe we should gate it behind `#[cfg(feature = "unstable_raw")]`.